keystrokes/src/components/utils/utils.js

```diff
@@ -1,5 +1,53 @@
 export function getNextKey(keys) {
-	return keys[
-		Math.floor(Math.random() * keys.length)
-	];
+	return keys[Math.floor(Math.random() * keys.length)];
+}
+
+export function verifyKey(key) {
+	if (key.length > 2) {
+		if (key === 'backquote') {
+			return '`';
+		} else if (key === 'bracketleft') {
+			return '{';
+		} else if (key === 'bracketright') {
+			return '}';
+		} else if (key === 'comma') {
+			return ',';
+		} else if (key === 'quote') {
+			return "'";
+		} else if (key === 'period') {
+			return '.';
+		} else if (key === 'slash') {
+			return '/';
+		} else if (key === 'backslash') {
+			return '\\';
+		} else if (key === 'equal') {
+			return '=';
+		} else if (key === 'minus') {
+			return '-';
+		} else if (key === 'semicolon') {
+			return ';';
+		}
+	}
+	return key;
+}
+
+export function validatePress(newKey) {
+	return (
+		newKey !== 'shiftleft' &&
```

Comment:
this is fine but an alternative could be an array of these strings and you could do `!keys.includes(newKey)`

Reply:
good call - swapped it